### DISCUSSION TITLE
How to reduce the session time for the user in the backend?

### DISCUSSION THREAD

**Discussion by Webmasters-WPNGO** [wintercms/winter#1121] [Q&A]
I want the user to have a limited authorization time in the backend. The only thing I noticed was the cookie `admin_auth` for the administrator. They are limited to more than a year. The challenge is to have employees who edit the website. They are authorized with their credentials and additionally enter a one-time password. When an employee leaves, they can still stay in the backend and know the authorization details. I can't control access by entering a one-time password to such an employee. I need to throw it out of the backend once a day and login again. Ideally, I imagine that access can be passwordless.

**Comment by LukeTowers** [comment]
@WebProviderNGO why don't you delete the user when they no longer should have access? When their account has been disabled / deleted they won't be able to access the backend regardless of whatever cookies they may have.

For the specific issue that you're asking for, take a look at `config/cms.php` -> `backendForceRemember`. You'd want to set it to false or null.
